lib/org/chromium/webidl/Canonicalizer.js

```diff
@@ -112,6 +131,51 @@ foam.CLASS({
       },
     },
     {
+      documentation: `Determines if the two Enum definitions are joint sets. If
+          the sets are disjoint (i.e. there is a member that exists in one but
+          not the other), we emit an warning and null is placed as entry in
+          Canonical definitions. Otherwise, the superset is placed into
+          Canonical definitions.`,
+      name: 'enumCmp_',
+      code: function(left, right) {
+        // Work on clones of definitions.
+        var leftClone = left.clone();
+        var rightClone = right.clone();
+        var leftMem = leftClone.definition.members;
+        var rightMem = rightClone.definition.members;
+        var id = left.id; // Name of definition is same for left and right.
+
+        // Allow the left hand side to always be the potential superset.
+        if (leftMem.length < rightMem.length) {
+          var tmp = leftMem;
```

Comment:
This and the check at the end could be eliminated if you did |this.enumCmp_(right, left)| here.